schema/index.go

```diff
@@ -138,3 +139,23 @@ func (m *MongoDBIndexer) IndexSymptomReportCollection() error {
 		},
 	})
 }
+
+func (m *MongoDBIndexer) IndexCDSConfirmCollection() error {
+	cdsIndex := mongo.IndexModel{
+		Keys:    bson.D{{"name", 1}, {"report_ts", 1}},
+		Options: options.Index().SetUnique(true),
+	}
+	usCollection := CDSCountyCollectionMatrix[CDSCountryType(CdsUSA)]
+	if err := m.createIndex(usCollection, cdsIndex); err != nil {
+		return err
+	}
+	twCollection := CDSCountyCollectionMatrix[CDSCountryType(CdsTaiwan)]
+	if err := m.createIndex(twCollection, cdsIndex); err != nil {
+		return err
+	}
+	icelandCollection := CDSCountyCollectionMatrix[CDSCountryType(CdsIceland)]
+	if err := m.createIndex(icelandCollection, cdsIndex); err != nil {
+		return err
+	}
```

Comment:
nit: using a for loop could be more concise.

Reply:
Could I ask what is `nit`?